mixer/test/client/rbac_permissive_global/rbac_permissive_global_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_test
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/istio/mixer/test/client/env"
+)
+
+// Report attributes from a GET request for Rbac global permissive mode.
+const reportAttributes = `
+{
+	"check.cache_hit": false,
```

Comment:
To indent, we use two spaces instead of tab. Please make it consistent with existing tests.

Reply:
done